src-opam/dockerfile_windows.ml

```diff
@@ -20,6 +20,13 @@ open Printf
 
 let run_cmd fmt = ksprintf (run "cmd /S /C %s") fmt
 let run_powershell fmt = ksprintf (run {|powershell -Command "%s"|}) fmt
+let run_vc ~arch fmt =
+  let arch = match arch with
+    | `I386 -> "x86" | `X86_64 -> "amd64"
+    | `Aarch64 | `Aarch32 | `Ppc64le -> failwith "Cross-compiling isn't supported."
```

Comment:
Cross-compiling is the wrong message here - it's an unsupported architecture. I have built OCaml on ARM64 Windows 10 using those tools - it's not cross-compilation!